src/controller/InputHandler.js

```diff
@@ -0,0 +1,35 @@
+import Validation from './Validation';
+
+class InputHandler {
+  getValidatedPurchseAmount(amount) {
+    Validation.validateNotEmpty(amount);
+    const purchseAmount = Number(amount);
+    Validation.validateIsNumber(purchseAmount);
+    Validation.validateThousandUnit(purchseAmount);
+    return purchseAmount;
+  }
```

Comment:
불필요한 라인을 줄이는 것도 좋지만.. 저는 공백도 코드라고 생각합니다!
아래와 같이 기준을 두어 분리해 두는 것도 고려해보시면 좋을 것 같아요!
```suggestion
  getValidatedPurchseAmount(amount) {
    Validation.validateNotEmpty(amount);
    
    const purchseAmount = Number(amount);
    Validation.validateIsNumber(purchseAmount);
    Validation.validateThousandUnit(purchseAmount);
    
    return purchseAmount;
  }
```